source/uri-options/uri-options.rst

```diff
@@ -282,17 +282,23 @@ pertaining to URI options apply here.
      - The name of the replica set to connect to
 
    * - retryReads
-     - "true" or "false
+     - "true" or "false"
      - defined in `retryable reads spec <https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#retryreads>`_
      - no
      - Enables retryable reads on server 3.6+
 
    * - retryWrites
-     - "true" or "false
+     - "true" or "false"
      - defined in `retryable writes spec <https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst#retrywrites>`_
      - no
      - Enables retryable writes on server 3.6+
 
+   * - sdamMode
```

Comment:
While "SDAM" trips easily off the tongue for all of us, it's not commonly understood by others.  Let's consider other names, e.g. "serverMonitoringMode".

Reply:
I agree. `serverMonitoringMode` is more descriptive than `sdamMode`.

Reply:
Agreed. I've renamed it "serverMonitoringMode".